### PR TITLE
Minor code structure adjustments to the bundles.js file

### PR DESCRIPTION
**Pull request by kunukn**
Purpose:
Consistent code in build.js and bundles.js.

build.js destructured bundleTypes, I think bundles.js should too.

yarn prettier on the bundles.js moved stuff around.

Added deepfreeze for bundleTypes and moduleTypes

**Before submitting a pull request,** please make sure the following is done:

1. Fork [the repository](https://github.com/facebook/react) and create your branch from `master`.
2. Run `yarn` in the repository root.
3. If you've fixed a bug or added code that should be tested, add tests!
4. Ensure the test suite passes (`yarn test`). Tip: `yarn test --watch TestName` is helpful in development.
5. Run `yarn test-prod` to test in the production environment. It supports the same options as `yarn test`.
6. If you need a debugger, run `yarn debug-test --watch TestName`, open `chrome://inspect`, and press "Inspect".
7. Format your code with [prettier](https://github.com/prettier/prettier) (`yarn prettier`).
8. Make sure your code lints (`yarn lint`). Tip: `yarn linc` to only check changed files.
9. Run the [Flow](https://flowtype.org/) typechecks (`yarn flow`).
10. If you haven't already, complete the CLA.

**Learn more about contributing:** https://reactjs.org/docs/how-to-contribute.html

### CODE REVIEW

File: scripts/rollup/bundles.js [hunk 1]
```diff
@@ -434,19 +434,16 @@ const bundles = [
 
   /******* ESLint Plugin for Hooks (proposal) *******/
   {
```
Comment (ahtee) [inline scripts/rollup/bundles.js]: any reason for this change? looks like it was intended to fit under the 
`/******* ESLint Plugin for Hooks (proposal) *******/` title

Comment (kunukn) [reply scripts/rollup/bundles.js]: Not from me. 
I executed `yarn prettier` as peer pull request guidance and prettier decided to do that.

**Comment by kunukn** [comment]
> looks great. does this fix any issue opened? can get it merged faster if so

I don't think so. It was something I stumbled on as I looked around in the source code.

**Comment by ahtee** [comment]
> I don't think so. It was something I stumbled on as I looked around in the source code.

Ok it'll be up to maintainers obviously if this will get merged then. I suppose the tests ran ok as well when you ran them?

**Comment by kunukn** [comment]
Yes the test ran ok

**Pull request merged**